scripts/gff_metaparser/genmetaconf/metaconf.py

```diff
@@ -244,6 +253,18 @@ def update_derived_data(self, defaults=None, update_annotation_related=False):
             syns.append((sci_name_words[0][0] + sci_name_words[1][:5]).lower())
         if syns and self.add_generated_species_aliases:
             self.update("species.alias", syns)
+        # picking assembly.alt_accession
+        if not self.get("assembly.alt_accession"):
+            asm_acc_insdc = self.get("assembly.accession_insdc")
+            asm_acc_refseq = self.get("assembly.accession_refseq")
+            if asm_acc_refseq and asm_acc_insdc:
+                if "refseq" in _ann_source.lower() or \
+                       "refseq" in self.get("annotation.provider_url").lower():
```

Comment:
What does this check do? I would think the annotation does not matter for the `assembly.*accession*` meta keys

Reply:
if there's /refseq/i in url

Reply:
We use this logic only for annotations from RefSeq annotation.
In theory, we can have genbank (insdc) assembly, that have a corresponding (GCF) RefSeq annotation, but a non-refseq annotation was loaded

Reply:
I believe we will only set the accession_refseq if the genome is from RefSeq so that check should not be necessary. It's fine to keep this although you need to make sure the annotation.source of provider_name are set properly

Reply:
thank you. keeping, though removing url checking bit